src/static/css/footer.less

```diff
@@ -240,7 +240,7 @@
         }
     }
 
-    .list_link__disabled {
```

Comment:
Hmm... good point. I forgot about those. In retrospect, I don't love naming utility mixins that way, but don't think we want to go changing that now.